README.md

```diff
@@ -1,4 +1,4 @@
-# Abstraction Layer or Ephemerides (ALE)
+# Abstraction Layer of Ephemerides (ALE)
```

Comment:
It should be Abstraction Layer for Ephemerides :X

Reply:
I saw this a while ago, but was confused about what name we actually settled on...